packages/next/server/dev/hot-reloader.ts

```diff
@@ -414,6 +420,55 @@ export default class HotReloader {
       )
   }
 
+  private async getVersionInfo(span: Span) {
+    const versionInfoSpan = span.traceChild('get-version-info')
+    return versionInfoSpan.traceAsyncFn<VersionInfo>(async () => {
+      try {
+        // TODO: Get the correct values
+
+        const installed = require('next/package.json').version
+        const installedParsed = semver.parse(installed)
+
+        const res = await fetch('TODO')
```

Comment:
Working on it.

Reply:
@styfle found a hidden API that returns exactly what we need, straight from npm. :relieved: https://registry.npmjs.org/-/package/next/dist-tags